mantidproject/mantid
# Workbench: Replace zoom icons in plot and slice-viewer GUI

**Description of work.**
Changed Mantid Workbench zoom icons to ``mdi.magnify`` in plot and slice-viewer toolbars. 

**To test:**
1. Plot a spectrum and ensure magnifying glass (zoom) icon does not contain a plus. 
2. Open the slice viewer for some data and check zoom icon does not contain a plus. 

Resolves #25837 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
